interop-testing/src/main/java/io/grpc/testing/integration/AbstractInteropTest.java

```diff
@@ -1732,6 +1745,37 @@ public void getServerAddressAndLocalAddressFromClient() {
     assertNotNull(obtainLocalClientAddr());
   }
 
+  /**
+   *  Test backend metrics reporting: expect the test client LB policy to receive load reports.
+   */
+  public void testOrca() throws Exception {
+    AtomicReference<OrcaLoadReport> reportHolder = new AtomicReference<>();
+    OrcaLoadReport answer = OrcaLoadReport.newBuilder()
+        .setCpuUtilization(0.1976)
+        .setMemUtilization(0.8712)
+        .putAllUtilization(ImmutableMap.of("util", 0.2938))
+        .build();
+    Payload data = Payload.newBuilder().setBody(answer.toByteString()).build();
```

Comment:
We shouldn't abuse the body like this. Orca is a completely different part of the message. What was wrong with an Any?